sdk/documentintelligence/Azure.AI.DocumentIntelligence/src/TrainingOperation.cs

```diff
@@ -11,21 +10,23 @@ namespace Azure.AI.DocumentIntelligence
 {
     internal class TrainingOperation : Operation<BinaryData>
     {
+        private const string OperationIdNotFoundErrorMessage = "The operation ID was not present in the service response.";
+
         private readonly Operation<BinaryData> _internalOperation;
 
-        private readonly Lazy<string> _operationId;
+        private readonly string _operationId;
```

Comment:
We were lazily extracting the operation ID from the response before, but this doesn't work for the endpoint we're trying to support now. The operation ID is returned only in the first POST response, so we need to extract it as soon as we can.